net/server.go

```diff
@@ -31,10 +31,11 @@ import (
 // returning accepted connections via a channel while handling
 // temporary network errors. Fatal errors are returned via the
 // error channel with the listener closed on return
-func StartAcceptLoop(l net.Listener, retrier xretry.Retrier) (<-chan net.Conn, <-chan error) {
```

Comment:
Can you add another constructor that simply takes `l net.Listener` that does the forever loop by default?  i.e. 
```go
func StartForeverAcceptLoop(l net.Listener) (<-chan net.Conn, <-chan error) {
  return StartAcceptLoop(l, xretry.NewOptions().SetForever(true))
}
``` 

That way users when looking at the package don't need to know they need to set a special flag, they can just use the constructor out of the box.